### PR TITLE
Cart Management

### PR DESCRIPTION
**Pull request by ritu9898**

### CODE REVIEW

File: components/styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  background-image: linear-gradient(-90deg, #A3278F 0%, #E52F50 100%);
+  color: white;
+  cursor: pointer;
+  border: 0;
+  font-size: 20px;
+}
+
+#bill-details {
+  box-shadow: 0 1px 8px 0 rgb(0 0 0 / 6%);
+  color: black;
```
Comment (Shubhamkr2610) [inline components/styles/cart.css:114]: use color with variable name

File: components/styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  margin: 1rem 0;
+  gap: 1rem;
+  padding-bottom: 1rem;
+  position: sticky;
+  bottom: 0;
+  border-top: 1px solid rgb(240, 240, 240);
+  background: rgb(255, 255, 255);
+  padding: 1rem 2rem;
+  box-shadow: rgb(0 0 0 / 10%) 0px -2px 10px 0px;
+}
```
Comment (Shubhamkr2610) [inline components/styles/cart.css:101]: use color with variable name

File: components/styles/cart.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  border: 1px solid grey;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.minus-disable {
+  color: #c2c2c2;
+  border-color: #e0e0e0;
+  cursor: auto !important;
```
Comment (Shubhamkr2610) [inline components/styles/cart.css:74]: use color with variable name

File: components/styles/cart.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  width: 3rem;
+  border: 1px solid grey;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.minus-disable {
+  color: #c2c2c2;
+  border-color: #e0e0e0;
```
Comment (DivyanshSareen) [inline components/styles/cart.css:73]: can use variables for colors

**Review by Shubhamkr2610 — APPROVED** [review]
Made the change that I have suggested , Apart from that you are good to go

**Review by DivyanshSareen — APPROVED** [review]
use variables for color hex-codes, everything else looks good to merge

File: components/styles/cart.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+.bill p span {
+  float: right;
+}
+
+.free-delivery {
+  color: forestgreen;
```
Comment (humzahasan) [inline components/styles/cart.css:136]: Suggestion : use variable for colors., as defined in CL

File: components/styles/cart.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  margin-top: 1rem;
+}
+
+.product-cart .add-more .minus, .product-cart .add-more .plus {
+  height: 2rem;
+  width: 2rem;
+  border-radius: 50%;
+  text-align: center;
+  border: 1px solid grey;
+  display: flex;
```
Comment (humzahasan) [inline components/styles/cart.css:52]: Suggestion : Add a seperate utility class for frequently used property.

File: components/styles/cart.css (hunk 5 above)
Comment (ritu9898) [reply components/styles/cart.css:136]: I've used different colors in different branches, and thus will change them to variables in final merge.

File: components/styles/cart.css (hunk 4 above)
Comment (ritu9898) [reply components/styles/cart.css:73]: I've used different colors in different branches, and thus will change them to variables in final merge.

**Review by sanu29 — APPROVED** [review]
good to merge

**Pull request closed without merging**